src/Microsoft.NetCore.Analyzers/CSharp/Security/CSharpDoNotUseInsecureCryptographicAlgorithms.cs

```diff
@@ -11,13 +11,7 @@ namespace Microsoft.NetCore.CSharp.Analyzers.Security
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public class CSharpDoNotUseInsecureCryptographicAlgorithmsAnalyzer : DoNotUseInsecureCryptographicAlgorithmsAnalyzer
     {
-        protected override SyntaxNodeAnalyzer GetAnalyzer(CompilationStartAnalysisContext context, CompilationSecurityTypes cryptTypes)
-        {
-            SyntaxNodeAnalyzer analyzer = new SyntaxNodeAnalyzer(cryptTypes);
-            context.RegisterSyntaxNodeAction(analyzer.AnalyzeNode,
-                                             SyntaxKind.InvocationExpression,
-                                             SyntaxKind.ObjectCreationExpression);
-            return analyzer;
-        }
+        // This analyzer just exists in the CSharp assembly so that people don't have to redo their rulesets,
```

Comment:
📝 The ruleset doesn't actually care what assembly and namespace you say the rule is in. It only looks at diagnostic ID and severity.

Reply:
Yes, that is correct. We should delete this file and move the `DiagnosticAnalyzer` attribute down to `DoNotUseInsecureCryptographicAlgorithmsAnalyzer`

Reply:
Oh!  Thanks, will remove these.